components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/client/authentication/OAuthClientAuthnServiceTest.java

```diff
@@ -199,14 +197,15 @@ public void testAuthenticateForFapiApplicationsWithInvalidAuthenticatorsRegister
         bodyParams.put(OAuth.OAUTH_CLIENT_ID, Arrays.asList(CLIENT_ID));
         PowerMockito.mockStatic(OAuth2Util.class);
         HttpServletRequest httpServletRequest = PowerMockito.mock(HttpServletRequest.class);
-        ServiceProvider serviceProvider = new ServiceProvider();
+        /*ServiceProvider serviceProvider = new ServiceProvider();
         ServiceProviderProperty fapiAppSpProperty = new ServiceProviderProperty();
         fapiAppSpProperty.setName(OAuthConstants.IS_FAPI_CONFORMANT_APP);
         fapiAppSpProperty.setValue("true");
-        serviceProvider.setSpProperties(new ServiceProviderProperty[]{fapiAppSpProperty});
+        serviceProvider.setSpProperties(new ServiceProviderProperty[]{fapiAppSpProperty});*/
```

Comment:
If this is not required lets remove this.

Reply:
removed